src/test/unit_tests/jest_tests/test_chunked_content_decoder.test.js

```diff
@@ -0,0 +1,259 @@
+/* Copyright (C) 2025 NooBaa */
+'use strict';
+
+const stream = require('stream');
+const assert = require('assert');
+const ChunkedContentDecoder = require('../../../util/chunked_content_decoder');
+const buffer_utils = require('../../../util/buffer_utils');
+
+describe('ChunkedContentDecoder', function() {
```

Comment:
need to add this test to any index file to include it in the jest tests? or does jest picks up all files from the dir?

Reply:
In jest, we add the tests (with the name convention `test_<file-that-we-test>.test.js`)  in the `jest` directory, and it runs in the CI.

I looked at the file in the CI for example and run:
`grep "no_chunk_with_trailers" /./0_run-unit-tests\ 4.txt | wc -l` and saw it is 200 (2 log printing in a test and 100 iterations).